packages/python3.rb

```diff
@@ -3,11 +3,11 @@
 class Python3 < Package
   description 'Python is a programming language that lets you work quickly and integrate systems more effectively.'
   homepage 'https://www.python.org/'
-  version '3.12.3'
+  version '3.12.4'
   license 'PSF-2.0'
   compatibility 'all'
   source_url "https://www.python.org/ftp/python/#{version}/Python-#{version}.tar.xz"
-  source_sha256 '56bfef1fdfc1221ce6720e43a661e3eb41785dd914ce99698d8c7896af4bdaa1'
+  source_sha256 'f6d419a6d8743ab26700801b4908d26d97e8b986e14f95de31b32de2b0e79554'
   binary_compression 'tar.zst'
 
   binary_sha256({
```

Comment:
We're missing binaries here?

Reply:
Also noticed filelist updates were missing too.

Reply:
Python is still building on my i686 builder.

Once that is done I will upload binaries and file lists...

Reply:
And only the ln will I take this PR out of draft. 😅

Reply:
Gotcha

Reply:
Done.